### PR TITLE
Default install and upgrade to "Y" when no answer is given

### PR DESCRIPTION
**Pull request by t-eckert**
Changes proposed in this PR:
- When a user is promped to install Consul, if they give no answer (e.g. just hit enter) the install will occur.

How I've tested this PR:
- Manually by running an install.

How I expect reviewers to test this PR:
- You may also run an install.

Checklist:
- [ ] Tests added
- [ ] CHANGELOG entry added 
  > HashiCorp engineers only, community PRs should not add a changelog entry.
  > Entries should use present tense (e.g. Add support for...)

### CODE REVIEW

File: cli/helm/install.go [hunk 1]
```diff
@@ -91,15 +91,15 @@ func InstallHelmRelease(options *InstallOptions) error {
...
+			Prompt: "Proceed with installation? (Y/n)",
 			Style:  terminal.InfoStyle,
 			Secret: false,
 		})
 
 		if err != nil {
 			return err
 		}
-		if common.Abort(confirmation) {
+		if confirmation != "" && common.Abort(confirmation) {
```
Comment (wilkermichael) [inline cli/helm/install.go:102]: I just suggest adding a comment on the intention here, that we intend to be approving on default

**Review by wilkermichael — APPROVED** [review]
Looks good! very minor comment

**Review by jmurret — COMMENTED** [review]
Should uninstall and upgrade follow the same paradigm of "you typed this command so I am going to default to you wanted to do this" or are we leaving those with N as the default because there is more risk to those?

**Comment by t-eckert** [comment]
> Should uninstall and upgrade follow the same paradigm of "you typed this command so I am going to default to you wanted to do this" or are we leaving those with N as the default because there is more risk to those?

I could see doing this for upgrade, but the convention is to only auto-yes for "additive" commands. I wouldn't do it for uninstall because it could be a footgun.

**Comment by david-yu** [comment]
Still waiting on @jmurret to review.

**Review by jmurret — APPROVED** [review]
looks good!  🙏

**Pull request merged**